.npmrc

```diff
@@ -1 +1,2 @@
-@backbase:registry=https://repo.backbase.com/api/npm/npm-backbase/
+# @backbase:registry=https://repo.backbase.com/api/npm/npm-backbase/
+@backbase:registry=https://artifacts.backbase.com/api/npm/npm-backbase/
```

Comment:
Can we merge this before v7 is on repo?

Reply:
an RC of ui-ang v8 was just promoted to repo, AFAIK that is the only reason why we needed artifacts here...

Reply:
They promoted it, so I reverted this